### PR TITLE
Update misc.py/ added web screenshot feature

### PR DESCRIPTION
**Pull request by Sachit71**
I added the web screenshotter feature, able to merge.

### CODE REVIEW

**Comment by Sachit71** [comment]
@aaryanrr closed and reopened the pr.

**Comment by codewithpom** [comment]
@Sachit71 Add typings to your functions

**Comment by Sachit71** [comment]
Oh I completely missed that part out, hold on.

File: pywhatkit/misc.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def take_screenshot(
         screen = ImageGrab.grab()
         screen.show(title=file_name)
         screen.save(f"{file_name}.png")
-def web_scr(link):
+def web_scr(link:str):
```
Comment (codewithpom) [inline pywhatkit/misc.py]: add ```-> None```

**Review by codewithpom — APPROVED** [review]
ok done

**Comment by aaryanrr** [comment]
Thanks for your contribution :)

**Comment by Sachit71** [comment]
You're welcome 😅

**Pull request merged**